skeleton_scary_game.py

```diff
@@ -0,0 +1,329 @@
+import random
+
+
```

Comment:
You should declare global variables here for all of the item string constants, the same way you did in the test file.